Tasks/Npm/npmtask.ts

```diff
@@ -1,5 +1,8 @@
+import fs = require('fs');
 import path = require('path');
+import url = require('url');
 import tl = require('vsts-task-lib/task');
+import tr = require('vsts-task-lib/toolrunner');
```

Comment:
I've used a pattern of trm elsewhere.  m meaning module.  tr being an instance of the class created from the module.  So

```
import trm = require('vsts-task-lib/toolrunner');
let tr : trm.ToolRunner = new trm.ToolRunner('foo');
```

typing tr makes it clear and it's also clear one is the module and the other is an instance of the class created from that module.